src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Answers {
+
+    private List<Answer> answers = new ArrayList<>();
+
+    public Answers() {
+    }
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public void deleteAnswers(NsUser nsUser) throws CannotDeleteException {
+        for (Answer answer : answers) {
+            answer.delete(nsUser);
+        }
+    }
```

Comment:
> 이런 경우에도 Answers 클래스에서 deleteAnswers 메서드에 대한 단위 테스트를 하는게 좋을까요?? 아니면 Answer에서 진행했기 때문에 따로 하지 않아도 괜찮을까요??

저는 public메소드이기 때문에 필요하다고 생각합니다.
내부 구현체가 변경되면서 기존의 로직에 영향을 끼칠수 있을수도 있기때문에 필요하다고 생각합니다.

```suggestion
public void deleteAnswers(NsUser nsUser) throws CannotDeleteException {
    어떠한 로직이 추가되면서 기존의 제약조건을 깨지게 만든다면?
    
    for (Answer answer : answers) {
        answer.delete(nsUser);
    }
}
```

Reply:
피드백 감사합니다!!

말씀하신 내용 참고해서 deleteAnswers에 대한 테스트도 추가했습니다.